mp/src/game/client/viewpostprocess.cpp

```diff
@@ -1502,11 +1502,11 @@ static void Generate8BitBloomTexture( IMatRenderContext *pRenderContext, float f
 	int nSrcHeight = pSrc->GetActualHeight(); //,dest_height;
 
 	// Counter-Strike: Source uses a different downsample algorithm than other games
-	#ifdef CSTRIKE_DLL
+	//#ifdef CSTRIKE_DLL
 		IMaterial *downsample_mat = materials->FindMaterial( "dev/downsample_non_hdr_cstrike", TEXTURE_GROUP_OTHER, true);
-	#else
-		IMaterial *downsample_mat = materials->FindMaterial( "dev/downsample_non_hdr", TEXTURE_GROUP_OTHER, true);
-	#endif
+	//#else
+	//	IMaterial *downsample_mat = materials->FindMaterial( "dev/downsample_non_hdr", TEXTURE_GROUP_OTHER, true);
+	//#endif
```

Comment:
We can remove the old code, we're on version control and can always see history of files.

Can we make that `downsample_non_hdr_cstrike` material instead be local? I have a feeling it's pulling it from the cstrike vpk and I'd like for it to be local, and we can even use the old name (`downsample_non_hdr`) for it. If I recall correctly it's just the `$cstrike 1` guy that makes the difference?

Reply:
The material is actually in the hl2 .vpks, oddly enough...